scipy/sparse/_base.py

```diff
@@ -63,6 +63,7 @@ class _sparray:
     """
 
     __array_priority__ = 10.1
+    _format = 'und'  # undefined
```

Comment:
What do we gain by this abbreviation? Can we not just use
```suggestion
    _format = 'undefined'
```
? AFAICT this should only ever be in the base class and thus not visible to the users anyway.

Reply:
I had that change originally, but then I saw that there is a dictionary of formats higher up in the file.

Reply:
OK, thx.